docs/start/getting-started/fragments/android/integrate.md

```diff
@@ -40,6 +40,23 @@ First, we'll add the DataStore plugin and configure Amplify. Typically, a good p
   </amplify-block>
   </amplify-block-switcher>
 
+You may run into this error:
```

Comment:
This is already directed on the first page of the tutorial, right? https://docs.amplify.aws/start/getting-started/setup/q/integration/android#add-amplify-to-your-application